src/makeStyles.tsx

```diff
@@ -37,7 +37,7 @@ export function createMakeStyles<Theme>(params: { useTheme: () => Theme }) {
             cssObjectByRuleNameOrGetCssObjectByRuleName:
                 | ((
                       theme: Theme,
-                      params: Params,
+                      params?: Params,
```

Comment:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#optional-parameters-in-callbacks